installer/conf/omsagent.d/mongo.conf

```diff
@@ -0,0 +1,37 @@
+<source>
+  type mongostat
+  tag oms.mongo
+  host localhost
+  port 27017
+  run_interval 5
+</source>
+
+#for mongoDB version < 3.0
+#<source>
+# type tail
+# path /var/log/mongodb/mongod.log
+# tag  oms.api.MongoDBlog
+# format /(?<timestamp>[^ ]*) \[(?<context>[^\]]*)\] (?<message>.*)/
+#</source>
+
+#for mongoDB version 3.0+
+<source>
+  type tail
+  path /var/log/mongodb/mongod.log
+  tag  oms.api.MongoDBlog
+  format /(?<timestamp>[^ ]*) (?<severity>[A-Z]) (?<component>[^ ]*)  \[(?<context>[^\]]*)\] (?<message>.*)/
+</source>
+
+<match oms.api.**>
+  type out_oms_api
+  log_level info
+
+  buffer_chunk_limit 5m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/out_oms_mongo*.buffer
```

Comment:
to avoid dup prefix, can we name the buffer file as out_oms_api_workload_mongo*.buffer?